dev_requirements.txt

```diff
@@ -0,0 +1 @@
+pytest
```

Comment:
it's better practice to pin these but it doesn't really matter for a project like this in my opinion

Reply:
yeah honestly we're not doing anything wacky with pytest and I can't be bothered to deal with pytest versions lol. I'll deal with it if it ever breaks, but I doubt it'll ever matter

Reply:
haha, agree 👍🏻